Antidote/ChatViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ChatViewController.swift
+//  Antidote
+//
+//  Created by Yoba on 02/01/2017.
+//  Copyright © 2017 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+
+class ChatViewController: NSViewController {
+    let chatView = ChatView()
+    var friend: OCTFriend
+    
+    init(friendObject friend: OCTFriend) {
```

Comment:
Just `init(friend: OCTFriend) {` would be enough. Less typing/reading is better in Swift.